doc/quickref.rst

```diff
@@ -427,6 +427,37 @@ Cmap mode
 | Y                    | Restore the color map to default ('gray').        |
 +----------------------+---------------------------------------------------+
 
+.. _plot_mode:
+
+Plot mode
```

Comment:
There is a note in the quick ref about how that mode can only be used with a viewer showing a plot with a `PlotAide`. The aide sets up that mode and puts the viewer in it.  The viewer can only be in one mode at a time (or none) so it will overwrite any mode the user set up before passing it in to `PlotAide`.